TLM/TLM/UI/Helpers/TrafficRulesOverlay.cs

```diff
@@ -138,19 +138,15 @@ private struct SignsLayout {
                 if (viewOnly_) {
                     // Readonly signs look grey-ish
                     guiColor = Color.Lerp(guiColor, Color.gray, 0.5f);
-                    guiColor.a = TrafficManagerTool.GetHandleAlpha(hovered: false);
                 } else {
-                    // Handles in edit mode are always visible. Hovered handles are also highlighted.
-                    guiColor.a = 1f;
-
                     if (hoveredHandle) {
                         guiColor = Color.Lerp(
                             a: guiColor,
                             b: new Color(r: 1f, g: .7f, b: 0f),
                             t: 0.5f);
                     }
                 }
-                // guiColor.a = TrafficManagerTool.GetHandleAlpha(hoveredHandle);
+                guiColor.a = TrafficManagerTool.GetHandleAlpha(hoveredHandle);
```

Comment:
Considering this is per-sign, we should consider calculating the various transparency values once, when main UI transparency is altered / loaded, rather than per sign. I'll create separate issue detailing that.

Reply:
👍